RPIPs/RPIP-42.md

```diff
@@ -1,113 +0,0 @@
-- If an NO has more total bonded ETH in their megapool than would be necessary based on the current settings (eg, `reduced_bond` is reduced), it SHALL be possible to reduce their bonded ETH and receive ETH `credit` for it
-- `credit` MUST be usable to create validators in a megapool
-- `credit` MUST be usable to mint rETH to the NO's primary withdrawal address 
-- The initial settings SHALL be:
-  - `base_bond_array`: [4, 8]
-  - `reduced_bond`: 4 ETH
-
-### Deposit queue specification
-ETH from the deposit pool SHALL be matched with validator deposits from queues as follows:
-- There SHALL be a `standard_queue`
-  - When adding a validator, users MAY place their deposit on the `standard_queue`  
-- There SHALL be an `express_queue`
-  - When adding a validator, users MAY place their deposit on the `express_queue` by spending one `express_queue_ticket`
-- When matching ETH from the deposit pool to queued deposits:
-  - First, ETH SHALL be matched to the oldest deposit in the `express_queue`; this is repeated `express_queue_rate` times
-  - Next, ETH SHALL be matched to the oldest deposit in the `standard_queue`
-  - This sequence SHALL be repeated indefinitely until both queues are empty
-    - If one queue is empty, those matches SHALL be skipped
-- Each node SHALL be provided `express_queue_tickets` equal to `express_queue_tickets_base_provision` (this includes newly created nodes)
-- Each node SHALL be provided additional `express_queue_tickets` equal to `(bonded ETH in legacy minipools)/4` (this will always be zero for newly created nodes)
-- The initial settings SHALL be:
-  - `express_queue_rate`: 2
-  - `express_queue_tickets_base_provision`: 2
-
-### Deposit mechanics specification
-- A node operator MUST take 3 actions to start a validator: `deposit`, `prestake`, and `stake`
-- `Deposit` SHALL place all the ETH in the deposit pool (where it can be used in validators as needed) and enters a queue as described [above](#deposit-queue-specification)
-- Until `prestake`, it SHALL be possible to exit the node operator queue and receive ETH `credit` for it
-- `Prestake` SHALL be possible when
-  - The deposit pool contains 31*`number of currently prestaked validators` ETH
-  - The deposit pool contains an additional 1 ETH per match using the process described [above](#deposit-queue-specification)
-- `Prestake` SHALL take 1 ETH from the deposit pool and stake it to the beacon chain
-- If there is enough ETH in the deposit pool to execute `deposit` and `prestake` in the same transaction, they MUST be done in the same transaction
-- There SHALL be a period of time to check for deposit validity
-- `Stake` SHALL take 31 ETH from the deposit pool and stake it to the beacon chain alongside the previous 1 ETH to make a complete validator
-
-## Specification taking effect with Saturn 2
-- Update `reduced_bond` to 1.5 ETH
-
-
-## Penalizable offenses
-This portion of the RPIP SHALL be considered Living. It may be updated by a DAO vote.
```

Comment:
Imo: "This portion of the RPIP SHALL be considered Living. It may be updated by a DAO vote following the existing rules and conventions for RPIP modifications."